packages/api/.nycrc.json

```diff
@@ -7,7 +7,7 @@
   "checkCoverage": true,
   "reporter": "html",
   "reportDir": "reports/coverage",
-  "exclude": [
-    "testResources/**/*.js"
+  "include": [
```

Comment:
what happened there? Exclude no longer needed?

Reply:
No, we should have used an `include` all along. I noticed it because of [this failing pipeline](https://github.com/stryker-mutator/stryker-js/runs/2521651082). Coverage was calculated on the _unit test files_ themselves, instead of the source files 😅

We're also using an `include` in all of our `.nycrc.json` files, don't know why not here.